### DISCUSSION TITLE
数据库版支持修改用户头像吗？

### DISCUSSION THREAD

**Discussion by m0eak** [lobehub/lobehub#5038] [General]
似乎没有找到相关功能

**Comment by SpeedupMaster** [comment]
如果配置了webhooks的话，可以更改logto和casdoor账户的头像，会更新到lobechat。如果用的clerk的话可以直接在lobechat上改

**Comment by JoeChen2me** [comment]
需要新增加一个数据库的桶用于存储头像，同时在鉴权服务中进行设置。这个你可以查看进阶部署中的`Docker Compose`部署这个文档。恰其中就有涉及。
